### DISCUSSION TITLE
why is the data being pulled into analytics only for yesterday?

### DISCUSSION THREAD

**Discussion by valery-lavrik** [invertase/react-native-firebase#5710] [General]
Why is there no data for today?

**Comment by valery-lavrik** [comment]
But Crashlytics works in real time

**Reply by mikehardy** [reply]
Analytics also works real time in debug view. If you want data for today, since the statistics are not correct they make you *explicitly* select today. This is a general google analytics question and should be posed on stack overflow for more general help - it is not specific to this module at all and is documented for google analytics, since it is not intuitive
